tools/dataProcessor.js

```diff
@@ -184,6 +193,15 @@ function parseDeals() {
       expiry_date: decode(xml_line.match("class=\"date\">(.*)<\\/div>")[1])
     };
 
+    // Removing unnecessary deal suffixes
+    let suffix_check = ['.com',' - Special',' - Featured','®','™' ]
```

Comment:
Do you think we should make this a separate function that works for both `deal` and `cashback`? This would reduce the duplicated code. Something like

```
function removeSuffixes(item) {
    ...your new code here...
}
```

And then you could move this entire section of code into there and just pass in a `deal` or `cashback` object.

If you prefer having two separate copies of the code, just apply the same comments from the `deal` section to here. Up to you which approach you want to take!

Reply:
I think the `removeSuffixes` function sounds interesting, I'll give that one a shot!